### ISSUE TITLE
Incompatibility with Django 1.7b

### ISSUE THREAD

**Issue by humitos** [edoburu/django-fluent-comments#43]
I'm getting this error:

```
Operations to perform:
  Synchronize unmigrated apps: facebook_comments, allauth, flatpages, chap, django_extensions, impersonate, sponsor, dbtemplates, fluent_comments, sites, contenttypes, crispy_forms, django_comments, django_proofread, sessions, reversion, auth, disqus, inspiration, league, account, admin, challenge, debug_toolbar, django_wysiwyg, team, organization, constance
  Apply all migrations: easy_thumbnails
Synchronizing apps without migrations:
  Creating tables...
  Installing custom SQL...
  Installing indexes...
Running migrations:
  No migrations needed.
Traceback (most recent call last):
  File "/home/humitos/Source/chap/chap/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 427, in execute_from_command_line
    utility.execute()
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 419, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/core/management/base.py", line 337, in execute
    output = self.handle(*args, **options)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 141, in handle
    changes = autodetector.changes(graph=executor.loader.graph)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/db/migrations/autodetector.py", line 36, in changes
    changes = self._detect_changes()
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/db/migrations/autodetector.py", line 53, in _detect_changes
    old_apps = self.from_state.render(ignore_swappable=True)
  File "/home/humitos/.virtualenvs/chap/local/lib/python2.7/site-packages/django/db/migrations/state.py", line 68, in render
    raise InvalidBasesError("Cannot resolve bases for %r" % new_unrendered_models)
django.db.migrations.state.InvalidBasesError: Cannot resolve bases for [<django.db.migrations.state.ModelState object at 0x7f0ac398f350>]

...

ipdb> new_unrendered_models[0]
<django.db.migrations.state.ModelState object at 0x7f702e039350>
ipdb> new_unrendered_models[0].name
u'FluentComment'
ipdb>

```

If I remove django-fluent-comments from my INSTALLED_APPS the error dissappears.

**Comment by Fingel** [comment]
I have this same problem

**Comment by vdboor** [comment]
Fixed! A django-fluent-comments 1.0 release has just been uploaded to PyPI with Django 1.7/1.8 support
